database/migrations/sql/20180604_06_create_users.sql

```diff
@@ -9,12 +9,30 @@ CREATE TABLE gender (
 );
 
 
+CREATE TABLE disability (
+  disability_id   SERIAL NOT NULL UNIQUE,
+  disability_name VARCHAR(100) NOT NULL UNIQUE,
+
+  CONSTRAINT disability_pk PRIMARY KEY (disability_id)
+);
+
+
+CREATE TABLE ethnicity (
+  ethnicity_id   SERIAL NOT NULL UNIQUE,
+  ethnicity_name VARCHAR(100) NOT NULL UNIQUE,
+
+  CONSTRAINT ethnicity_pk PRIMARY KEY (ethnicity_id)
+);
+
+
 CREATE TABLE user_account (
   user_account_id                  SERIAL NOT NULL UNIQUE,
   user_name                        VARCHAR(100) NOT NULL,
   user_password                    VARCHAR(64),
   qr_code                          VARCHAR UNIQUE,
   gender_id                        INT NOT NULL,
+  disability_id                    INT NOT NULL,
```

Comment:
Is this having a default?

Reply:
I think no, because it's an ID, and I don't want to rely on particular IDs corresponding to particular values. We'll enforce the default in application code.

Reply:
Out of curiosity, why are we using id's rather than enums?